### ISSUE TITLE
Power state is always on when audio output set to homepod 

### ISSUE THREAD

**Issue by Shkaufma** [postlund/pyatv#1667]
### Describe the bug

When setting the audio output of the Apple TV to a homepod (and not tv speakers), a query to get device power state is always Powerstate.On

when changing back to audio output to tv speakers then the state reports are accurate

### How to reproduce the bug?

Set audio output to homepod (I used mini)
Try to get device state, it will show Powerstate.On
Power AppleTv to standby mode
Read the power state - it is still Powerstate.On

### What is expected behavior?

When apple tv is off it is expected to report Powerstate.off

### Operating System

TvOs 15.3

### Python

3.7

### pyatv

0.10.0

### Device

Apple TV 4k 2nd gen

### Additional context

None

**Comment by postlund** [comment]
Yep, this is a known problem and I don't have any ways to fix it unfortunately.

**Comment by postlund** [comment]
@Shkaufma  Is this still the case in the latest HA release?

**Comment by alkalixin** [comment]
Same issue here.
I'm using HA 2024.1.6
